homepage/views.py

```diff
@@ -234,25 +238,31 @@ def homepage(request, template_name="homepage.html"):
         .order_by("?")[:5]
     )
 
-    # try:
-    #     potw = Dpotw.objects.latest().package
-    # except Dpotw.DoesNotExist:
-    #     potw = None
-    # except Package.DoesNotExist:
-    #     potw = None
-    #
-    # try:
-    #     gotw = Gotw.objects.latest().grid
-    # except Gotw.DoesNotExist:
-    #     gotw = None
-    # except Grid.DoesNotExist:
-    #     gotw = None
+    try:
+        potw = (
+            Dpotw.objects.filter(start_date__lte=my_today, end_date__gte=my_today)
+            .latest()
+            .package
+        )
+    except Dpotw.DoesNotExist:
+        potw = None
+    except Package.DoesNotExist:
+        potw = None
+
+    try:
+        gotw = (
+            Gotw.objects.filter(start_date__lte=my_today, end_date__gte=my_today)
+            .latest()
+            .grid
+        )
+    except (Gotw.DoesNotExist, Grid.DoesNotExist):
+        gotw = None
 
     # Public Service Announcement on homepage
-    # try:
-    #     psa_body = PSA.objects.latest().body_text
-    # except PSA.DoesNotExist:
-    #     psa_body = '<p>There are currently no announcements.  To request a PSA, tweet at <a href="http://twitter.com/open_comparison">@Open_Comparison</a>.</p>'
+    try:
+        psa_body = PSA.objects.latest().body_text
+    except PSA.DoesNotExist:
+        psa_body = '<p>There are currently no announcements.  To request a PSA, tweet at <a href="http://twitter.com/open_comparison">@Open_Comparison</a>.</p>'
```

Comment:
Interesting, did not know something like this existed. Is `@Open_Comparison` twitter account active anymore?

Reply:
@saadmk11 I'm not sure. This was code that was already in the file that had been commented out. I put it back in with the intention of asking @jefftriplett what, if any, method of communication may be set up for this kind of thing. I'm not sure if there's an official DjangoPackages presence on either X or Mastodon. 

This may be something that can be tied into Django News potentially, but that's just an idea I was kind of kicking around.

Reply:
I created https://fosstodon.org/@djangopackages and have used it sparingly to mainly communicate downtime. 

I'd be fine just not having a PSA if there isn't anything to shout out. That's probably the easiest way to manage it. 

(I'll check this out later tonight or the first of the week)

Reply:
I carved out some time to check this out today and I quickly realized I need to develop a better plan for how to roll out the Grids and Packages. I'll work on this over the weekend and see which grids can be cleaned up to start targeting. I can do a follow-up about announcements too. 

@ryancheley thank you for getting this ink on the page aka championing these features to bring them back.